Stripe/PublicHeaders/STPSourceParams.h

```diff
@@ -49,25 +49,25 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  A set of key/value pairs that you can attach to a source object.
  */
-@property (nonatomic, copy, nullable) NSDictionary *metadata;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *,NSString *>*metadata;
 
 /**
  *  Information about the owner of the payment instrument. May be used or required
  *  by particular source types.
  */
-@property (nonatomic, copy, nullable) NSDictionary *owner;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *, id>*owner;
 
 /**
  *  Parameters required for the redirect flow. Required if the source is authenticated by 
  *  a redirect (`flow` is "redirect").
  */
-@property (nonatomic, copy, nullable) NSDictionary *redirect;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *, id>*redirect;
```

Comment:
These hashes can contain inner hashes (e.g. owner["address"]). I think that's the only example of an inner hash right now, but it seems better to be more permissive for everything except metadata